components/motor/gpio/motor_encoder.go

```diff
@@ -102,15 +102,16 @@ func newEncodedMotor(
 	em.encoder = realEncoder
 
 	if len(motorConfig.ControlLoop.Blocks) != 0 {
-		cLoop, err := control.NewLoop(logger, motorConfig.ControlLoop, em)
-		if err != nil {
+		if err = em.FindControlBlock(); err != nil {
 			return nil, err
 		}
-		err = cLoop.Start()
+		pidBlock, err := em.loop.ConfigAt(cancelCtx, "PID")
 		if err != nil {
 			return nil, err
 		}
-		em.loop = cLoop
+		if pidBlock.Attribute["kP"] == 0 && pidBlock.Attribute["kI"] == 0 && pidBlock.Attribute["kD"] == 0 {
+			em.loop.Tuning = true
+		}
```

Comment:
does this freak out if they aren't set for a non-control loop motor?

Reply:
no because it won't get to this line if it's a non-controlled motor

Reply:
currently only have the control loop set up for the right motor on bucket and left still works normally with rpm monitor

Reply:
excellent test rig.